playbooks/collect-logs.yml

```diff
@@ -114,6 +118,36 @@
         segfault_problems_found: "{{ result.stdout_lines }}"
       ignore_errors: yes
 
+    - name: Collect logs from nova
+      sudo: yes
+      shell: cp -Rf /var/log/nova/* /var/log/extra/openstack
+      ignore_errors: yes
+
+    - name: Collect logs from glance
+      sudo: yes
+      shell: cp -Rf /var/log/glance/* /var/log/extra/openstack
+      ignore_errors: yes
+
+    - name: Collect logs from cinder
+      sudo: yes
+      shell: cp -Rf /var/log/cinder/* /var/log/extra/openstack
+      ignore_errors: yes
+
+    - name: Collect logs from keystone
+      sudo: yes
+      shell: cp -Rf /var/log/keystone/* /var/log/extra/openstack
+      ignore_errors: yes
+
+    - name: Collect logs from neutron
+      sudo: yes
+      shell: cp -Rf /var/log/neutron/* /var/log/extra/openstack
+      ignore_errors: yes
+
+    - name: Collect logs from horizon
+      sudo: yes
+      shell: cp -Rf /var/log/httpd/horizon_error.log /var/log/extra/openstack
+      ignore_errors: yes
+
```

Comment:
Collect-logs is based on grabbing everything in `{{ job.archive }}` list see [link](https://github.com/rhosqeauto/InfraRed/blob/a3bbb4448bade115b08aaa8ddc2dc566fe13e5d8/playbooks/collect-logs.yml#L132-L140). 

Please according to [`ospd.yml`](https://github.com/rhosqeauto/InfraRed/blob/9471828d8bf4088f9920216650afb996d733f39f/settings/installer/ospd/ospd.yml#L13-L13) the entire `/var/log/*` tree should be archived in all nodes.

Please investigate why this doesn't apply for undercloud node